librz/bin/bin.c

```diff
@@ -221,7 +221,22 @@ RZ_API void rz_bin_symbol_free(RzBinSymbol *sym) {
 	free(sym);
 }
 
-RZ_API void rz_bin_reloc_free(RzBinReloc *reloc) {
+/**
+ * \brief Free \p reloc
+ * 
+ * \param reloc RzBinReloc instance to free
+ * \return void
+ */
+RZ_API void rz_bin_reloc_free(RZ_NONNULL RzBinReloc *reloc) {
+	rz_return_if_fail(reloc);
```

Comment:
no, actually just write `if (!reloc) { return; }`
free methods MUST accept NULL.

Reply:
`rz_return_if_fail` is like an assert.